eng/common/scripts/Rest-Methods.ps1

```diff
@@ -0,0 +1,124 @@
+. "${PSScriptRoot}\logging.ps1"
+
+function Invoke-RestMethodPost {
```

Comment:
I'm not sure what these functions buy us above the built in ```Invoke-RestMethod```.

Reply:
I'm also curious about the same question.  Do we think the having these helpers is worth it?

Reply:
It does give a central place for handling errors that might occur so we don't have to do that all over the place.